CharlieBackend.Api/Controllers/MentorsController.cs

```diff
@@ -160,19 +158,35 @@ public async Task<IList<MentorCoursesDto>> GetMentorCoursesByMentorId(long id)
         }
 
         /// <summary>
-        /// Disabling of mentor account
+        /// Disable mentor
         /// </summary>
-        /// <response code="204">Successful disabling of mentor</response>
-        /// <response code="HTTP: 400, API: 3">Can not find mentor</response>
+        /// <response code="204">Mentor successfully disabled</response>
+        /// <response code="HTTP: 400, API: 3">Mentor not found</response>
+        /// <response code="HTTP: 409, API: 5">Mentor is already disabled</response>
```

Comment:
IMO a message like "_Mentor's **account** is already disabled_" would be more accurate,
instead of saying that someone is disabled. ♿ ♿︎

Reply:
Solved in each controller i've changed